### ISSUE TITLE
Movement of keyboard focus from the 'big play' button to the full control bar to support keyboard accessibility

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#2729]
A couple of changes would fix accessibility issues related to the 'big play' button:
1. When the big play button is activated using the keyboard, **focus should be moved** to the regular play button in the control bar
2. There should be some indication to screen reader users that only the big play button is available before playback is started, and that the full control bar will become available when play starts (or that it has become available once the big play button has been activated, but this is not an ideal solution because that is the point when audio from the video playback will start).

This is related to, but distinct from. #2156.

**Comment by chrisauclair** [comment]
Can we expect a fix for this before 6.0?

**Comment by gkatsev** [comment]
@chrisauclair yeah, I think that this change can definitely be made without a major release. PRs are always welcome but otherwise, it may be a while before this lands.
